generator/Dockerfile.tt

```diff
@@ -6,43 +8,42 @@ FROM ruby:$RUBY_VERSION-slim-$DISTRO_NAME
 ARG DISTRO_NAME
 
 # Common dependencies
-RUN apt-get update -qq \
-  && DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends \
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Comment:
Can we add a link explain what `--mount` is here? (Link to this PR will work, I think)

Reply:
Done, added a link to the Dockerfile reference